src/handlers/poster-hubble/components/Clock.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+
+interface Props {
+  color: string;
+}
+
+export default function Clock({ color }: Props) {
+  const [hours, setHours] = useState<number | undefined>();
+  const [minutes, setMinutes] = useState<number | undefined>();
+
+  const setTime = () => {
+    const now = new Date();
+    setHours(now.getHours());
+    setMinutes(now.getMinutes());
+  };
+
+  useEffect(() => {
+    const interval = setInterval(setTime, 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, []);
+
+  return (
+    <div
+      className="inline-flex flex-row flex-nowrap w-auto h-full items-center"
+      style={{ fontFamily: '"Lato", monospace', fontSize: '8vh', color: color }}
+    >
+      <div>{hours?.toString().padStart(2, '0') ?? '--'}</div>
+      <div className="w-full text-center" style={{ width: '=10%' }}>
+        :
+      </div>
+      <div>{minutes?.toString().padStart(2, '0') ?? '--'}</div>
+    </div>
+  );
+}
```

Comment:
Is the Hubble clock so different from the GEWIS clock that it needs its own component? Is there no way to reuse the existing clock and parameterize it more to be able to create the Hubble clock?

Reply:
The clock did change quite a bit for how simple the clock is. Furthermore I think for this one, and also most of the others, that is easier to just leave them as seperate components than to create switches and parameters for every little thing. Especially for maintability, then new Hubble people can just make pull request changing the HTML+CSS of the overlay, instead of having to do all sort of complicated things with making sure it switches correctly based on if it is for Hubble or for GEWIS etc etc. It is code duplication but in favor of readability and maintability

Reply:
Two main things were 
- Clock also needs to be colourable 
- GEWIS clock does not nicely scale to 4K screens

Reply:
But also small things like no more shadows, no more flickering of the ":" etc.